inst/shiny-server/server/ERTPlot.R

```diff
@@ -24,7 +24,7 @@ update_ert_per_fct_axis <- observe({
 
 render_ert_per_fct <- reactive({
   withProgress({
-  req(input$ERTPlot.Min, input$ERTPlot.Max, DATA())
+  req(input$ERTPlot.Min, input$ERTPlot.Max, length(DATA()) > 0)
```

Comment:
I think `req` will check if the length of DATA() is zero (maybe I am wrong). But, this is a minor thing...

Reply:
req() uses the isTruthy function, which does give False when applied to NULL, but gives True when used on an empty DataSetList object, which is why I added the check on length.